cime_config/machines/cmake_macros/crayclang.cmake

```diff
@@ -1,11 +1,14 @@
 if (compile_threaded)
-  string(APPEND CFLAGS " -h omp")
-  string(APPEND FFLAGS " -h omp")
-  string(APPEND LDFLAGS " -h omp")
+  string(APPEND CFLAGS   " -fopenmp")
```

Comment:
i did not try to run scaling studies to see if either flag works with OMP, but from googling I thought -homp is the correct flag fro cray (but that flag throws an error if used with hipcc).

Reply:
Why do you want to run scaling studies to check this flag?

In the case of Cray Fortran, both flags are equivalent. Since Cray C compiler is Clang based, the OpenMP flag is `-f openmp`

From the Cray Fortran man page
```
>       -f [no]openmp
              Enable  or  disable  compiler  recognition of OpenMP directives.  Using -fnoopenmp is similar to the -h
              thread0 option, in that it disables OpenMP, but unlike -h  thread0,  -fnoopenmp  does  not  affect  au-
              tothreading.   This  option is an alias to -h[no]omp.  CrayPE will link in the serial version of LibSci
              when -fnoopenmp is used.

              � Default: -f noopenmp [-h noomp]
```

Reply:
thanks, Sarat. i ran into an error only with `hipcc -homp`. Re: scaling -- i think it is good to actually confirm that OMP works (and flags, too), the only way I know how to do it is to run a code that scales (homme, for example). But this is out of range of my current tasks, so i just made a note about it.

Reply:
so, great to know that -fopenmp should work, too.

Reply:
Don't we have a small case which can be used to check MPI vs. MPI+OMP perf?

Reply:
As I mentioned in other comments, some updates are needed for things in this PR. So scaling runs as is is not recommended.